modules/alloy/src/main/ts/ephox/alloy/positioning/layout/Layout.ts

```diff
@@ -1,8 +1,13 @@
+import { Fun } from '@ephox/katamari';
+import { Bounds } from '../../alien/Boxes';
 import { nu as NuSpotInfo } from '../view/SpotInfo';
 import { Bubble } from './Bubble';
 import * as Direction from './Direction';
+import { adjustBounds, anchorBottom, anchorLeft, anchorRight, anchorTop } from './LayoutBounds';
 import { AnchorBox, AnchorElement, AnchorLayout } from './LayoutTypes';
 
+const identity = Fun.identity;
```

Comment:
`identity` probably isn't the easiest name to identify as keep the bounds position. Although the function is called `adjustBounds`, so I guess it's reasonable to assume `identity` would just preserve the bounds. The other options you have are something a bit more usage-specific like `preserveBound` or even the duplicated: `keepLeft`, `keepRight`. There are arguments for all approaches.

Reply:
Actually, going through it a bit more ... I'm starting to understand. So all the functions that are pre-defined except for identity, completely ignore `bounds`, so `identity` is just the one that uses `bounds`.

I wonder if that would be clearer if it was `useBounds` vs `useRightAnchor` or something similar. Obviously, wasting time discussing names isn't particularly important. So perhaps just a comment or two as to the intent of `identity` vs the `anchor*` adjustment functions (because they aren't really adjusting bounds ... they are just replacing)